src/cheri_insts.sail

```diff
@@ -1430,6 +1430,84 @@ function clause execute (LoadCapCap(cd, cs1)) = {
   handle_load_cap_via_cap(cd, 0b0 @ cs1, cs1_val, vaddr)
 }
 
+union clause ast = CLoadTags : (regidx, regidx)
+/*!
+ * Integer register *rd* is replaced with the tags of the capabilities located
+ * at and above *cs1*.**address**.  The 0th bit corresponds to the first
+ * capability in memory.  The result is coherent with other processors, *as if*
+ * the corresponding data words had also been loaded.  The number of tags
+ * loaded is implementation defined; typical implementations are expected to
+ * return the tags held in a L1 cache line, and so we use the sail constant
+ * *caps_per_cacheline*.  The number of tags loaded must be a power of two, at
+ * least 1, and no more than **XLEN**.
+ *
+ * ## Exceptions
+ *
+ * An exception is raised if:
+ *   - *cs1*.**tag** is not set.
+ *   - *cs1* is sealed.
+ *   - *cs1* does not grant both **Permit_Load** and
+ *     **Permit_Load_Capabilities**.
+ *   - *cs1*.**address** $\lt$ *cs1*.**base**.
+ *   - *cs1*.**address** $+$ **caps_per_cacheline* $*$ **CLEN** $/$ 8 $\gt$ *cs1*.**top**.
+ *   - *cs1*.**address** is not properly aligned.
+ *   - The page mapping for *cs1*.**address** specifies that capability loads are to
+ *     to trigger traps or clear tags.
```

Comment:
```suggestion
 *   - The page table entry for *cs1*.**address** would cause the tag to be cleared.
```
since the other traps are normal for loads (and we don't document the MMU faults in every load/store instruction, since they come from the MMU)